app/assets/javascripts/discourse/app/components/bread-crumbs.js

```diff
@@ -10,11 +10,11 @@ export default Component.extend({
   editingCategory: false,
   editingCategoryTab: null,
 
-  @discourseComputed("category.ancestors", "categories", "noSubcategories")
-  categoryBreadcrumbs(categoryAncestors, filteredCategories, noSubcategories) {
-    categoryAncestors = categoryAncestors || [];
-    const parentCategories = [undefined, ...categoryAncestors];
-    const categories = [...categoryAncestors, undefined];
+  @discourseComputed("category", "categories", "noSubcategories")
+  categoryBreadcrumbs(cat, filteredCategories, noSubcategories) {
+    const ancestors = cat?.ancestors || [];
```

Comment:
🚫🐱 😄 

```suggestion
  categoryBreadcrumbs(category, filteredCategories, noSubcategories) {
    const ancestors = category?.ancestors || [];
```

Reply:
I named it like this because there is another category defined a few lines down and I would get a warning about shadowed variables. Maybe I should rename the other one.